lib/errors.js

```diff
@@ -5,10 +5,12 @@ const codes = {
   NOT_FOUND: 1001,
   INVALID_SCHEMA: 1002,
   INVALID_CATEGORY_TYPE: 1003,
+  NO_SUCH_ACTION: 1004,
 };
 
 const defaultMessages = {
   [codes.NOT_IMPLEMENTED]: 'Not implemented',
+  [codes.NO_SUCH_ACTION]: 'No such action',
```

Comment:
I am against adding a default message for this case, I think you should always specify the action that is missing.